skills/roborazzi/references/gradle_properties_options.md

```
@@ -0,0 +1,73 @@
+<!-- Generated from docs/topics/gradle_properties_options.md by the generateSkill task. Do not edit this file. -->
+# Roborazzi gradle.properties Options and Recommendations
+
+You can configure the following options in your `gradle.properties` file.
+You can also use `-P` to set the options in the command line. For example, `./gradlew test -Proborazzi.test.record=true`.
+
+## roborazzi.test
+
+This option enables you to configure the behavior of Roborazzi. By default, all settings are set to false.
+For additional configuration options, please refer to the 'Apply Roborazzi Gradle Plugin' section.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)